### DISCUSSION TITLE
How to build an app

### DISCUSSION THREAD

**Discussion by aleksandr-vin** [Lennolium/swiftGuard#4] [Questions and Answers] [answered]
It would be beneficial to have apart from [build-dmg.sh](https://github.com/Lennolium/swiftGuard/blob/main/build-dmg.sh) also some sort of _build-app.sh_ that one can run and get an apple app.

**Comment by Lennolium** [comment]
Thanks for your suggestion, I will add such a .sh file today.

**Comment by Lennolium** [comment, marked as answer]
> It would be beneficial to have apart from [build-dmg.sh](https://github.com/Lennolium/swiftGuard/blob/main/build-dmg.sh) also some sort of _build-app.sh_ that one can run and get an apple app.

I have completely redone the [create-dmg.sh](https://github.com/Lennolium/swiftGuard/blob/main/build-dmg.sh) and created a [build-app.sh](https://github.com/Lennolium/swiftGuard/blob/main/build-app.sh). Now you just need to executed both scripts and get a complete build.

Hope you like them! 😊

I'll close this discussion in a few days. Feel free to open another discussion or issue, if you have further suggestions/help.
